### PR TITLE
Fix undefined error for `ActionController::Parameters`

### PR DESCRIPTION
**Pull request by maclover7**
This was introduced as apart of the 9-CVE-push (aka 5.0.0.beta1.1).

r? @tenderlove

### CODE REVIEW

**Comment by maclover7** [comment]
This will also need to be backported to 4-2-stable.

File: actionpack/lib/abstract_controller/rendering.rb [hunk 1]
```diff
@@ -82,13 +82,10 @@ def view_assigns
     # <tt>render :file => "foo/bar"</tt>.
     # :api: plugin
     def _normalize_args(action=nil, options={})
-      case action
-      when ActionController::Parameters
-        unless action.permitted?
-          raise ArgumentError, "render parameters are not permitted"
-        end
+      if action.respond_to?(:permitted?)
```
Comment (prathamesh-sonpatki) [inline actionpack/lib/abstract_controller/rendering.rb]: Doesn't this change behavior? We are now raising if action respods to `permitted?` not *if* action is permitted or not?

**Comment by prathamesh-sonpatki** [comment]
Can this undefined error be not fixed by just adding `require 'action_controller/metal/strong_parameters'` in this file?

**Comment by vipulnsward** [comment]
This worked for me- https://gist.github.com/vipulnsward/3fc7cf7c51d60f6c4e6c
@prathamesh-sonpatki I think the purpose would be to keep AC isolated from Metal.

**Comment by maclover7** [comment]
@prathamesh-sonpatki - we are trying to avoid coupling on AbstractController to ActionController -- it is meant to be the other way around. Yes a require would fix the issue, but we also need to remove the underlying coupling that's going on here.

**Pull request merged**